internal/ethapi/api.go

```diff
@@ -1358,8 +1423,17 @@ func DoEstimateGas(ctx context.Context, b Backend, args TransactionArgs, blockNr
 			if transfer == nil {
 				transfer = new(hexutil.Big)
 			}
+			maxFeeInFeeCurrency := args.MaxFeeInFeeCurrency
+			if maxFeeInFeeCurrency == nil {
+				maxFeeInFeeCurrency = new(hexutil.Big)
+			}
+			feeCurrency := "<nil>"
+			if args.FeeCurrency != nil {
+				feeCurrency = args.FeeCurrency.String()
+			}
```

Comment:
Is this necessary? I would expect that directly logging `args.FeeCurrency` is readable.